site/zenodo_rdm/assets/semantic-ui/js/zenodo_rdm/src/support/components/CategoryDropdown.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+import { SelectField } from "react-invenio-forms";
+import _find from 'lodash';
+import _isEmpty from 'lodash';
+
+class CategoryDropdown extends Component {
+
+    constructor(props) {
+        super(props);
+
+        let activeCategory;
+        const { categories, defaultCategory } = props;
+
+        if (!_isEmpty(categories)) {
+            activeCategory = categories.find((el) => (el.key === defaultCategory))
```

Comment:
minor: You can use the [?](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator for simplifying the code maybe
```suggestion
            activeCategory = categories?.find((el) => (el.key === defaultCategory))
```